sql/core/src/main/scala/org/apache/spark/sql/sources/interfaces.scala

```diff
@@ -216,10 +216,13 @@ abstract class BaseRelation {
 
   /**
    * Whether does it need to convert the objects in Row to internal representation, for example:
+   *
+   * {{{
    *  java.lang.String -> UTF8String
    *  java.lang.Decimal -> Decimal
+   * }}}
```

Comment:
Likewise just use something besides "->" to mean "to", instead of a code block to work around